### DISCUSSION TITLE
Cant get data from Vnish S19

### DISCUSSION THREAD

**Discussion by bugy855** [UpstreamData/pyasic#105] [Q&A] [answered]
Hello, I am trying to write a simple script to get data from one miner

```
import asyncio 
from pyasic.network import MinerNetwork
from pyasic import get_miner

async def get_data_from_miner():
    miner = await get_miner("192.168.10.48")
    data = await miner.get_data()
    print(data)

if __name__ == "__main__":
    asyncio.run(get_data_from_miner())
```

When I run this script I get this error

Traceback (most recent call last):
  File "C:\.venv\Lib\site-packages\pyasic\miners\base.py", line 437, in _get_data
    miner_data[data_name] = await function(**args_to_send)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\.venv\Lib\site-packages\pyasic\miners\backends\vnish.py", line 211, in _get_fw_ver
    fw_ver = fw_ver.split("(Vnish ")[1].replace(")", "")
             ~~~~~~~~~~~~~~~~~~~~~~~^^^
IndexError: list index out of range

Did I do something stupid here? I followed the getting started page of the docs and can't seem to get past this.

**Comment by UpstreamData** [comment]
Nope, this is a bug.  Not your issue.  Ill fix this and push an update for you.

**Comment by UpstreamData** [comment, marked as answer]
Fixed in 0.50.8, please update and try again :)

**Reply by bugy855** [reply]
Works great, thank you for the quick response and fix!
